kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/IfcfgConfigWriter.java

```diff
@@ -52,15 +50,20 @@
     private static final String REDHAT_NET_CONFIGURATION_DIRECTORY = "/etc/sysconfig/network-scripts/";
     private static final String DEBIAN_NET_CONFIGURATION_FILE = "/etc/network/interfaces";
     private static final String DEBIAN_TMP_NET_CONFIGURATION_FILE = "/etc/network/interfaces.tmp";
+    
+    private static final String LOCALHOST = "127.0.0.1";
 
-    private static IfcfgConfigWriter s_instance;
+    private static IfcfgConfigWriter instance;
+    
+    private static List<String> debianInterfaceComandOptions = new ArrayList<>(Arrays.asList("pre-up", "up", "post-up", "pre-down", "down", "post-down"));
+    private static List<String> debianIgnoreInterfaceCommands = new ArrayList<>(Arrays.asList("post-up route del default dev"));
 
     public static IfcfgConfigWriter getInstance() {
-        if (s_instance == null) {
-            s_instance = new IfcfgConfigWriter();
+        if (instance == null) {
```

Comment:
given that you re modifying this method...it should be made ``synchronized`` just to avoid concurrency issues.
Also update the end date in the copyright header for every file you change.